manifests/server/puppetserver.pp

```diff
@@ -0,0 +1,29 @@
+class puppet::server::puppetserver (
+  $java_bin          = $::puppet::server_jvm_java_bin,
+  $config            = $::puppet::server_jvm_config,
+  $jvm_min_heap_size = $::puppet::server_jvm_min_heap_size,
+  $jvm_max_heap_size = $::puppet::server_jvm_max_heap_size,
+  $jvm_extra_args    = $::puppet::server_jvm_extra_args,
+) {
+
+  $puppetserver_package = pick($::puppet::server_package, 'puppetserver')
+
+  Augeas {
+    lens    => 'Shellvars.lns',
+    incl    => $config,
+    context => "/files${config}",
+    notify  => Service['puppetserver'],
+    require => Package[$puppetserver_package],
```

Comment:
It might be better to anchor the class in puppet::server::config, so it always runs in that "context", which will always be after the install class has run.